exercicio_2/nataliatrogo

```diff
@@ -0,0 +1,44 @@
+#Exercícios para casa de Python. Números 8,11,12 e 14.
+
+print("Exercício 8:") #Ler um número real e dividi-lo em parte inteira e parte fracionária.
+
+num = float(input("Insira um número real: "))
+#O valor que o usuário fornece e que será retornada pelo input() é sempre um string e não um número. Antes de fazer a conta, precisamos converter para float.
+
+parte_int = int(num)
+
+# Para usar só dois dígitos decimais, usar round(número,num de decimais)
+parte_fr = round(float(num) - parte_int,2)
+
+
+print("A parte inteira é", parte_int," e a parte fracionária é ", parte_fr,'\n')
+
+
+#Exercício 11:Em um frigorífico existem 50 bois. Cada boi traz preso no pescoço um cartão contendo seu número de identificação e o peso. Elaborar um algoritmo para ler os dados de todos os bois e escrever somente o número e o peso do boi mais gordo. Admitir que todos os bois têm pesos diferentes.
+
+
+#num_bois = 50
+#cart_identif = input("Insira o Cartão de Visitação")
+
+print("Exercício 12:") #Elaborar um algoritmo para ler dez números e calcular a média aritmética dos números.
+
+num1 = float(input("Insira o primeiro número: "))
```

Comment:
Essa solução poderia ter sido feita dentro de um `for`. Na próxima aula vamos ver em detalhes a solução desses exercícios.

Segue uma sugestão:

```
def exercicio_12():
    soma = 0
    for i in range(10):
        num = float(input('Insira um número. '))
        soma += num

    media = soma / 10
    print('A média dos números inseridos é', media)
```